### DISCUSSION TITLE
Guess the song 🎸

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#734] [Galaxy Trivia] [answered]
```
E|-------5-7-----7-|-8-----8-2-----2-|-0---------0-----|--------------|
B|-----5-----5-----|---5-------3-----|---1---1-----1---|-0-1-1--------|
G|---5---------5---|-----5-------2---|-----2---------2-|-0-2-2--------|
D|-7-------6-------|-5-------4-------|-3---------------|--------------|
A|-----------------|-----------------|-----------------|-2-0-0---0----|
E|-----------------|-----------------|-----------------|--------------|
```

**Bonus:** Name the songwriter.

**Comment by peterbaccaro** [comment]
The song is **Seven Nation Army** by The White Stripes. The song was written and produced by **Jack White**.

**Comment by thuongtruong1009** [comment]
Based on the provided guitar tab, it appears to be the song "Wonderwall" by Oasis. The songwriter for "Wonderwall" is Noel Gallagher.

**Comment by SANKARAMDAS** [comment, marked as answer]
"Stairway To Heaven" by Led Zeppelin.
This is the really nice acoustic fingerstyle guitar intro of the famous song by the English rock band Led Zeppelin.

**Reply by ParanoidUser** [reply]
Bravo! 👏👏
